src/components/manage/Widgets/ArrayWidget.jsx

```diff
@@ -119,18 +137,33 @@ class ArrayWidget extends Component {
     super(props);
 
     this.handleChange = this.handleChange.bind(this);
+  }
+
+  getValue() {
```

Comment:
watch out! it's not binded!

Reply:
There's more wrong with that ArrayWidget... I'll fix it and then I think the PR is good for review.

Reply:
Ah... I've asked Silviu for some Cypress tests for this... I didn't get them yet. We need to wait for them.